system/jlib/jbuff.hpp

```diff
@@ -189,6 +188,8 @@ class jlib_decl MemoryBuffer
     inline void     Release() const                         { delete this; }    // for consistency even though not link counted
 
     inline void *   bufferBase() const { return buffer; }
+    inline const char * toByteArray() const { return curLen ? buffer : NULL; }
```

Comment:
use nullptr?

Reply:
code moved rather than new, but I will change it.